app/logic/serviceLearningCourses.py

```diff
@@ -77,17 +77,47 @@ def saveCourseParticipantsToDatabase(cpPreview: Dict[str, Dict[str, Dict[str, Li
                 print(f"Unable to save course {course}. {courseInfo['errorMsg']}")
                 continue
 
-            courseObj: Course = Course.get_or_create(
-                 courseAbbreviation = course,
-                 term = termObj, 
-                 defaults = {"CourseName" : "",
-                             "sectionDesignation" : "",
-                             "courseCredit" : "1",
-                             "term" : termObj,
-                             "status" : 4,
-                             "createdBy" : g.current_user,
-                             "serviceLearningDesignatedSections" : "",
-                             "previouslyApprovedDescription" : "" })[0]
+            termCheck = Term.get(Term.id == termObj)
+
+            checkTermOrder = termCheck.termOrder
```

Comment:
`termObj` is already the term object, and we know it exists, we can just use this. And we should name it something else, like `courseTerm`

Reply:
We changed "termCheck" to "currentTermObj" and "termObj" to "courseTerm"